File: app/src/main/java/com/futsch1/medtimer/reminders/scheduling/ScheduledReminder.java
package com.futsch1.medtimer.reminders.scheduling;

import com.futsch1.medtimer.database.FullMedicine;
import com.futsch1.medtimer.database.Reminder;

import java.time.Instant;

public record ScheduledReminder(FullMedicine medicine, Reminder reminder, Instant timestamp) {

}

File: app/src/main/java/com/futsch1/medtimer/reminders/scheduling/ScheduledReminder.kt
package com.futsch1.medtimer.reminders.scheduling

import com.futsch1.medtimer.database.FullMedicineEntity
import com.futsch1.medtimer.database.ReminderEntity
import java.time.Instant

data class ScheduledReminder(
    val medicine: FullMedicineEntity,
    val reminder: ReminderEntity,
    val timestamp: Instant
)
